libs/partners/ai21/langchain_ai21/chat_models.py

```diff
@@ -48,14 +63,14 @@ class ChatAI21(BaseChatModel, AI21Base):
     stop: Optional[List[str]] = None
     """Default stop sequences."""
 
-    max_tokens: int = 16
+    max_tokens: int = 512
     """The maximum number of tokens to generate for each response."""
 
     min_tokens: int = 0
     """The minimum number of tokens to generate for each response.
     _Not supported for all models._"""
 
-    temperature: float = 0.7
+    temperature: float = 0.4
```

Comment:
is this related to this pr?

Reply:
Hi @baskaryan, this isn't related directly to this pr, we've just decided internally to set the default temperature to 0.4 instead of 0.7.